app/templates/views/guidance/using-notify/attach-pages.html

```diff
@@ -0,0 +1,45 @@
+{% extends "content_template.html" %}
+
+{% from "components/service-link.html" import service_link %}
+
+{# Used by the content_template.html layout, prefixes the "navigation" accessible name #}
+{% set navigation_label_prefix = 'Using Notify' %}
+
+{% block per_page_title %}
+  Attach pages
+{% endblock %}
+
+{% block content_column_content %}
+
+  <h1 class="heading-large">Attach pages</h1>
+
+  <p class="govuk-body">Add attachments to your letters such as documents, tables and direct debit mandates.</p>
+  <p class="govuk-body">Upload a PDF and we’ll print it as part of your letter.</p>
+
+
+  <p class="govuk-body">To add an attachment to your letter:</p>
+
+  <ol class="govuk-list govuk-list--number">
+    <li>Go to the {{ service_link(current_service, 'main.choose_template', 'templates') }} page.</li>
+    <li>Add a new template or choose an existing template</li>
+    <li>Select <b class="govuk-!-font-weight-bold">Attach pages</b> and <b class="govuk-!-font-weight-bold">choose a file</b>.</li>
+  </ol>
+
+  <p class="govuk-body">Your PDF must meet our attachment specification.</p>
+
+  <h2 class="heading-medium">Attachment specification</h2>
+
+  <p class="govuk-body">Page size and layout: A4 portrait (210 × 297 mm)</p>
+  <p class="govuk-body">Maximum file size: 2 MB</p>
+  <p class="govuk-body">Your letter must be 10 pages or less (5 double-sided sheets of paper).</p>
+  <p class="govuk-body">The content of your letter must appear inside the printable area.</p>
+  <p class=govuk-body">The printable area is:</p>
+
```

Comment:
I don’t feel too strongly about this - but I wonder if we need the repetition or not?



```suggestion
  <p class="govuk-body">The content of your letter must appear inside the printable area:</p>

```